src/examples/rpp/doxygen/interval.cpp

```diff
@@ -0,0 +1,32 @@
+#include <rpp/rpp.hpp>
+
+#include <iostream>
+
+/**
+ * \example defer.cpp
+ **/
+
+int main() // NOLINT
+{
+    //! [interval period]
+    rpp::source::interval(std::chrono::milliseconds(10), rpp::schedulers::immediate{})
+        | rpp::operators::take(3)
+        | rpp::operators::subscribe([](size_t v) { std::cout << v << "\n"; }, rpp::utils::rethrow_error_t{}, []() { std::cout << "On complete\n"; });
```

Comment:
I think, in this case it is important WHEN v was obtained. Could you please add something like this?
https://github.com/victimsnino/ReactivePlusPlus/blob/v2/src/examples/rpp/doxygen/delay.cpp#L14C5-L14C53

Reply:
done